source/agent/addons/quic/QuicTransportStream.cc

```diff
@@ -0,0 +1,225 @@
+/*
+ * Copyright (C) 2020 Intel Corporation
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "QuicTransportStream.h"
+
+using v8::Function;
+using v8::FunctionTemplate;
+using v8::Local;
+using v8::Object;
+using v8::ObjectTemplate;
+using v8::Value;
+
+DEFINE_LOGGER(QuicTransportStream, "QuicTransportStream");
+
+Nan::Persistent<v8::Function> QuicTransportStream::s_constructor;
+
+const int uuidSizeInByte = 16;
```

Comment:
uuidSizeInBytes, instead of uuidSizeInByte

Reply:
Done.